doc/classes/ProjectSettings.xml

```diff
@@ -821,6 +821,14 @@
 			The default screen orientation to use on mobile devices. See [enum DisplayServer.ScreenOrientation] for possible values.
 			[b]Note:[/b] When set to a portrait orientation, this project setting does not flip the project resolution's width and height automatically. Instead, you have to set [member display/window/size/viewport_width] and [member display/window/size/viewport_height] accordingly.
 		</member>
+		<member name="display/window/hdr/enabled" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], enables HDR output on supported platforms, falling back to SDR if not supported.
+			Only available on platforms that support HDR output, have HDR enabled in the system settings, and have a compatible display connected.
+		</member>
+		<member name="display/window/hdr/max_luminance" type="float" setter="" getter="" default="0.0">
+			Sets the maximum luminance of the display in nits (cd/m²) when HDR is enabled.
```

Comment:
```suggestion
			Sets the maximum luminance of the display in nits (cd/m²) when HDR is enabled.  If set to [code]0.0[/code], luminance is not limited, which may look worse than setting a max luminance value suited to the display currently in use.
```

Reply:
Also not so sure about this, for the reasons in the other related comment.